packages/sn-filter-pane/src/qae/object-definition.js

```diff
@@ -13,12 +13,6 @@ const objectDefinition = () => {
      * @default
      */
     version: process.env.PACKAGE_VERSION,
-    /**
-     * Show visualization details toggle
-     * @type {boolean=}
-     * @default
-     */
-    showDetails: true,
```

Comment:
I think we do want to keep the default value in properties.
maybe you can mark it as private to remove it from the spec??